src/Data/Form.purs

```diff
@@ -642,6 +664,19 @@ mvpObjective =
       ]
     )
 
+mvpSocialAccount :: Field Expr
```

Comment:
I threw together this as a toggle because toggles are simple. It's not intended to be anything more than a simple mock up to allow us a second tab. Happy to change it to something else if we'd like.